postapiusingfile.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+
+var file = function (req, res, next) {
+    var data = fs.readFileSync("input.json")
+    var file = JSON.parse(data);
+    var stringfile= JSON.stringify(file.payload)
+    Object.keys(file.referenceData).forEach(key => {
+        var toReplace =`{${key}}`;
+        var regex = new RegExp(toReplace, "g");
+        stringfile = stringfile.replace(regex,file.referenceData[key])
```

Comment:
this code works for small size files.
can you check with 4MB json file? and mention the system load
On Windows, use task manager -> ram and cpu usage spikes.
On Linux, top command.
compare that with iterate json and replace version of the code cpu usage.

Reply:
Hi Arul,

I checked with the 4mb json file. you are right it will effect performance a;so i checked the ram and memory status from task manager. I changed my code by iterating through json object which give better performance than previous code

Thanks,
Amit